lib/reporters/base.js

```diff
@@ -544,7 +554,6 @@ var objToString = Object.prototype.toString;
 function sameType(a, b) {
   return objToString.call(a) === objToString.call(b);
 }
-
```

Comment:
[Style] Nit: could you undo this unrelated change & any others please?

Reply:
Will do